tools/proteore_get_unique_peptide_srm_method/proteore_get_unique_peptide_SRM-MRM_method.xml

```diff
@@ -0,0 +1,167 @@
+            <option value="file" selected="true">Input file containing Uniprot-AC identifiers</option>
+        </param>
+        <when value="text" >
+            <param name="txt" type="text" label="Copy/paste IDs" help='IDs must be separated by tab, space or carriage return into the form field, for example: P31946 P62258' >
+                <sanitizer invalid_char="">
+                    <valid initial="string.printable">
+                        <remove value="&apos;"/>
+                    </valid>
+                    <mapping initial="none">
+                        <add source="&apos;" target="__sq__"/>
+                        <add source="&#x20;" target=""/>
+                        <add source="&#xA;" target=""/>
+                        <add source="&#xD;" target=""/>
+                        <add source="&#x9;" target=""/>
+                    </mapping>
+                </sanitizer>
+            </param>
+        </when>
+        <when value="file" >
+            <param name="input_file" type="data" format="txt,tabular" label="Select your file" help="" />
+            <param name="header" type="boolean" checked="true" truevalue="true" falsevalue="false" label="Does file contain header?" />
+            <param name="ncol" type="text" value="c1" label="Column number of IDs" help='For example, fill in "c1" if it is the first column, "c2" if it is the second column and so on'>
+                <validator type="regex" message="Please enter a column number, for example: 'c1' for the first column">[c]{0,1}[0-9]+</validator>
+            </param>
+        </when>
+    </conditional>
+    <param name="srm_file" type="select" label="Release" >
+        <options from_data_table="proteore_human_srm_atlas">
+            <filter type="sort_by" column="0"/>
+        </options>
+    </param>
+    <param name="protein_features" type="select" label="Peptide sequence/features" multiple="true" help="" display="checkboxes" optional="false">
+        <option value="PeptideSeq" selected="true">Peptide sequence</option>
+        <option value="SSRT" selected="false">SSRT (Sequence Specific Retention Time)</option>
+        <option value="Length" selected="false">Length (peptide sequence length)</option>
+        <option value="MW" selected="false">MW (Molecular weight)</option>
+        <option value="PA_AccNum" selected="false">PeptideAtlas Accession (PA_Acc)</option>
+    </param>
+    </inputs>
+    <outputs>
+        <data name="output" format="tsv"/>
+    </outputs>
+    <tests>
+        <test>
+            <conditional name="input" >
+                <param name="ids" value="file"/>
+                <param name="input_file" value="Lacombe_et_al_2017_modified.tsv" />
+                <param name="header" value="true" />
+                <param name="ncol" value="c1"/>
+            </conditional>
+            <param name="srm_file" value="tool-data/Human_SRM_atlas_2016-04.csv.gz" />
```

Comment:
This is wired. I would assume the `value` needs to be the key/id from the datatable. E.g. `9979839599`?